packages/ember-htmlbars/tests/helpers/with_test.js

```diff
@@ -281,122 +32,3 @@ QUnit.module('{{#with}} helper binding to view keyword', {
 QUnit.test('{{with}} helper can bind to keywords with \'as\'', function() {
```

Comment:
So I'll leave the test as it's test how `with` aliasing into the block params and context up.